testing/integration_tests.py

```diff
@@ -953,6 +945,28 @@ def test_cd_bit_set(self):
         name = "dnssec-failed.org"
         cmd, res = self.run_zdns(c, name)
         self.assertSuccess(res, cmd)
- 
+
+    def test_timetamps(self):
+        c = "A"
+        name = "zdns-testing.com"
+        cmd, res = self.run_zdns(c, name)
+        self.assertSuccess(res, cmd)
+        assert "timestamp" in res
+        date = datetime.datetime.strptime(res["timestamp"], "%Y-%m-%dT%H:%M:%S%z")
+        self.assertTrue(date.microsecond == 0)  # microseconds should be 0 since we didn't call with --nanoseconds
+
+    def test_timetamps_nanoseconds(self):
+        c = "A --nanoseconds"
+        name = "zdns-testing.com"
+        cmd, res = self.run_zdns(c, name)
+        self.assertSuccess(res, cmd)
+        assert "timestamp" in res
+        date = datetime.datetime.strptime(res["timestamp"], "%Y-%m-%dT%H:%M:%S.%f%z")
+        self.assertTrue(date.microsecond != 0)
+        # microseconds should be non-zero since we called with --nanoseconds. There is a chance it happens to be 0,
+        # but it is very unlikely. (1 in 1,000,000). Python's datetime.date's smallest unit of time is microseconds,
+        # so that's why we're using this in place of nanoseconds. It should not affect the test's validity.
+
+
```

Comment:
Note to reviewers: apologies for the large change-set, my IDE auto-formatter decided to format the file according to PEP 8. These are the only functional, non-format changes